README.md

```diff
@@ -410,6 +411,22 @@ Four actions are available for you to hook onto through the tooltip/popover life
 }}
 ```
 
+## Test Helpers
+
+This addon exposes testing helpers which can be used inside of the consuming app's acceptance and integration tests. We use a tooltip-centric naming convention but these can also be used to test popovers.
+
+* `assertTooltipVisible($body, assert)`: asserts if the tooltip is visible. It checks two attributes: aria-hidden and data-tether-enabled.
+* `assertTooltipNotVisible($body, assert)`: asserts if the tooltip is not visible. It checks two attributes: aria-hidden and data-tether-enabled.
+* `assertTooltipRendered($body, assert)`: asserts if the tooltip has been rendered. When enableLazyRendering is true the tooltip will only be rendered after the user has interacted with the $target element.
```

Comment:
I'll update the documentation. This is different because a tooltip can be rendered but not visible.